tests/Doctrine/Tests/DBAL/Driver/Mysqli/MysqliConnectionTest.php

```diff
@@ -28,11 +28,31 @@ protected function setUp()
             ->getMockForAbstractClass();
     }
 
+    protected function tearDown()
+    {
+        restore_error_handler();
```

Comment:
This will cause `restore_error_handler()` to be called for every test in this class - that is a mistake, since any error handler registered by phpunit may be replaced. Only the actual test overriding the error handler should restore the error handler (by calling `restore_error_handler()` at the end)